infrastructure/oss/inline_value.go

```diff
@@ -0,0 +1,40 @@
+/*
+ * © 2023 Snyk Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package oss
+
+import (
+	"github.com/rs/zerolog/log"
+
+	"github.com/snyk/snyk-ls/domain/snyk"
+)
+
+func (oss *Scanner) GetInlineValues(path string, myRange snyk.Range) ([]snyk.InlineValue, error) {
+	method := "GetInlineValues"
+	log.Debug().Str("method", method).Str("path", path).Msg("called")
+
+	value := oss.inlineValues[path][myRange.Start.Line]
```

Comment:
Question: The current assumption here seems to me, that start line and end line are the same. Is this actually correct. In the lsp specification it says
> The document range for which inline values should be computed.

So I would be assuming that the range can span multiple lines and we would need to iterate over the inlineValues here to determine if they are in range or not.

Reply:
For OSS vulnerability count it's only the line that's relevant though.  That's why in general, we work with the range, but in the concrete implementation not.

Reply:
I understand that the OSS vulnerability count is associated to a single line. But isn't this logic a lookup/search functionality that is triggered by the lsp client and the lsp client doesn't know anything about this, it only known the viewport of a document to show.